docs/source/user-manual/blocks.md

```diff
@@ -0,0 +1,54 @@
+---
+myst:
+  html_meta:
+    "description": "This is the end-User documentation for Plone 6 Volto frontend."
+    "property=og:description": "Volto is the React-based frontend for the Plone CMS. It is the default UI for the Plone 6 release."
+    "property=og:title": "Frontend"
+    "keywords": "Volto, Plone, frontend, React, User manual"
+---
+
+```{todo}
+This page needs content
+```
+
+# Blocks
+Volto features the Pastanaga Editor Engine, allowing you to visually compose a page using blocks. The editor allows you to add, modify, reorder and delete blocks given your requirements. Blocks provide the user the ability to display content in an specific way, although they can also define behavior and have specific features.
+
+## Types of pre-built Blocks
+
+You can access and choose one block out of different types of pre-built blocks by:
```

Comment:
````suggestion
```{todo}
Add description of a Volto block.
```

Volto offers several default block types out of the box.
You can access and choose a block type to add to your content type when you have an empty block in it.

To create an empty block after an existing block, click in the block, then hit the {kbd}`Enter` key.
A new empty block appears.

```{todo}
Add screenshots or video illustrating the steps.
```

```{note}
There is a new experimental feature that places a `+` below every block in a content type.

See https://github.com/plone/volto/pull/3815 for details of the feature and how to enable it.
```

Now with your empty block available, you can select its type in one of two ways.
````